postprocessing/janus-pp-rec.c

```diff
@@ -1171,15 +1187,28 @@ int main(int argc, char *argv[])
 			JANUS_LOG(LOG_VERB, "[%10lu][%4d] time=%"SCNu64"s\n", tmp->offset, tmp->len, tmp->ts);
 		tmp = tmp->next;
 	}
-	JANUS_LOG(LOG_INFO, "Counted %"SCNu32" frame packets\n", count);
-	if(rotated != -1) {
-		if(rotated == 0 && last_rotation != 0) {
-			JANUS_LOG(LOG_INFO, "The video is rotated\n");
-		} else if(rotated > 0) {
-			JANUS_LOG(LOG_INFO, "The video changed orientation %d times\n", rotated);
+	if(!ext_only) {
+		JANUS_LOG(LOG_INFO, "Counted %"SCNu32" frame packets\n", count);
+		if (rotated != -1) {
+			if (rotated == 0 && last_rotation != 0) {
+				JANUS_LOG(LOG_INFO, "The video is rotated\n");
+			} else if (rotated > 0) {
```

Comment:
You changed the code style when you moved the code, please don't do that. Per our code style, it's `if(` and not `if (` (no space).